llm_bioinformatics_research/src/HomeScreen/HomeScreen.js

```diff
@@ -1,79 +1,96 @@
 import React from 'react';
-import { Container, Box, Typography, Paper, TextField, Grid } from '@mui/material';
+import { Container, Box, Typography, Paper, TextField, Grid, IconButton } from '@mui/material';
 import { grey } from '@mui/material/colors';
+import { Edit, Refresh, Upload, Send} from '@mui/icons-material';
+
+const CustomPaper = ({ children, title, height }) => (
+  <Paper
+    elevation={3}
+    sx={{
+      p: 2,
+      bgcolor: grey[300],
+      border: '1px solid',
+      borderColor: grey[400],
+      height: height || 'auto',
+      display: 'flex',
+      flexDirection: 'column',
+      flexGrow: height ? 1 : 0,
+    }}
+  >
+    {title && <Typography variant="h6">{title}</Typography>}
+    {children}
+  </Paper>
+);
+
+const ReadOnlyTextField = ({ placeholder, rows }) => (
+  <TextField
+    fullWidth
+    multiline
+    rows={rows}
+    variant="outlined"
+    placeholder={placeholder}
+    sx={{ mt: 2, flexGrow: 1, overflowY: 'auto' }}
+    InputProps={{
+      readOnly: true,
+    }}
+  />
+);
 
 
 function HomeScreen() {
   return (
     <>
-      <Container maxWidth="lg" sx={{ mt: 3, mb: 10 }}>
-        <Grid container spacing={2}>
-          <Grid item xs={12} md={6}>
-            <Paper
-              elevation={3}
-              sx={{
-                p: 2,
-                height: '100%',
-                bgcolor: grey[100],
-                border: '1px solid',
-                borderColor: grey[400],
-              }}
-            >
-              <Typography variant="h6">Code Snippet (Input)</Typography>
-              <TextField
-                multiline
-                fullWidth
-                rows={10}
-                placeholder="Enter code here..."
-                variant="outlined"
-                sx={{ mt: 2 }}
-              />
-            </Paper>
+      <Container maxWidth={false} sx={{ mt: 2, mb: 2, minHeight: '80vh', display: 'flex', flexDirection: 'column' }}>
+        {/*AI Chat and Response */}
```

Comment:
This is a general comment about the page sizing. I noticed that come of the css elements are done by pixel count. However, that will likely cause issues for different screen sizes. We might want to reformat these elements by the percentage of the screen (vertical and horizontal). 

For reference: 

![image](https://github.com/user-attachments/assets/e675dfca-57b1-4183-9cab-0b82142f5288)